src/utils/Transaction.ts

```diff
@@ -15,6 +15,7 @@ export function createAndReturnTransaction(event: ethereum.Event): Transaction {
     transaction.from = event.transaction.from.toHexString()
     transaction.to = event.transaction.to
     transaction.value = event.transaction.value
+    transaction.functionSignature = event.transaction.input.toHexString().slice(0, 10)
```

Comment:
why? for the swaps task?

Reply:
yes, and general data analysis

Reply:
using this is a discussion of its own, and this should be done in the relevant PR then

Reply:
I'll remove